From 7b4a712b3ad90fb39c075b63207c14e64db65506 Mon Sep 17 00:00:00 2001
From: Hugh Guan <hhguan@codeaurora.org>
Date: Mon, 12 Sep 2016 17:27:07 +0800
Subject: [PATCH 2/2] default.pa: Load acdb module

Automatically load acdb module for audio calibration
database loader of automotive platform.

Signed-off-by: Hugh Guan <hhguan@codeaurora.org>
---
 src/daemon/default.pa.in | 5 ++++++++
 1 file changed, 5 insertions(+)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index d157da7..afbd8b3 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -77,6 +77,11 @@ load-module module-udev-detect tsched=0
 load-module module-detect
 .endif
 
+### Automatically load acdb module by QTI
+.ifexists module-acdb@PA_SOEXT@
+load-module module-acdb
+.endif
+
 ### Automatically connect sink and source if JACK server is present
 .ifexists module-jackdbus-detect@PA_SOEXT@
 .nofail
-- 
1.9.1

